Sources/Mistica/Utils/Extensions/UIBarButtonItem+Utils.swift

```diff
@@ -47,3 +49,28 @@ extension UIButton {
         frameHeight = 36.0
     }
 }
+
+public extension UIButton {
+    /*
+     Support SVG Images from URL for UIButton settings
+     */
+    func setImageFromURL(url: URL, urlForDarkMode: URL? = nil) {
```

Comment:
```suggestion
    func setImage(from url: URL, darkURL: URL? = nil) {
```
I think this follows better the guidelines of swift, wdyt?

Reply:
Since it is a method expressly made for iphoneApp, there we are using urlDarkMode so it is better to keep it like this for similarity.